paasta_tools/cli/cmds/status.py

```diff
@@ -1492,3 +1570,15 @@ def status_marathon_job_human(
         return "Marathon:   {} - {} (app {}) is not configured in Marathon yet (waiting for bounce)".format(
             status, name, desired_app_id
         )
+
+
+# Add other custom status writers here
+# See `print_tron_status` for reference
+INSTANCE_TYPE_WRITERS: Mapping[str, InstanceStatusWriter] = defaultdict(
```

Comment:
nit: can we push this to the top of the file?

Reply:
@mattmb I tried pushing this to the top but python somehow is not able to resolve reference to these methods

Reply:
Ah, yeah, thanks for trying

python--